### DISCUSSION TITLE
Should it be permissible to kill a Mime that spoke, so long as the brain is preserved?

### DISCUSSION THREAD

**Discussion by TaM-tHe-JaM** [ParadiseSS13/Paradise#21925] [Documents]
Mimes don't talk. Everybody knows that. So whatever it is that spoke....

Well. It ain't a Mime, and it has to go. The brain can be transferred to a fixed body.

This is both legitimate and for lulz a recommendation I made and wanted to see people's opinions on. C'mon, it's Paradise. If there's no 'Mime Spoke' warrant, is it REALLY that mind boggling?

(Obviously I'm not demanding it, I just want to see people's thoughts on if they'd actually like it as a feature in the event it was, in fact, added.)

**Comment by TaM-tHe-JaM** [comment]
I know it needs labels but I'm a GitHub noob and I don't know how to add that.

**Comment by Kenionatus** [comment]
Way too LRP for Paradise in my opinion.
